docs/server/settings.rst

```diff
@@ -414,6 +414,19 @@ Translation environment configuration settings.
   The directory where the translation files are kept.
 
 
+.. setting:: QUALITY_CHECKER
+
+``QUALITY_CHECKER``
+  Default: ``''``
+
+  The import path to a funtion that provides alternate quality checks to
+  Pootle.  When left unset then the checking functions available from the
+  Translate Toolkit, and set in the project Admin page, are used instead.
+
+  .. note:: If set only that checker function is used and the checker funtions
+     from Translate Toolkit are not used, the two cannot be selectively applied.
```

Comment:
I believe you mean: *.. note:: If set, only the checker function defined here is used instead of the Translate Toolkit counterparts. Both cannot be selectively applied.*